doc/man/8/ceph-volume.rst

```diff
@@ -34,6 +36,25 @@ them.
 Commands
 ========
 
+inventory
+---------
+
+This subcommand provides information about a hosts physical disc inventory and
```

Comment:
Maybe:
```suggestion
This subcommand provides information about physical disc inventory in a host, and
```

Otherwise `hosts` should be `hosts's`